src/sage/categories/regular_crystals.py

```diff
@@ -842,13 +842,8 @@ def dual_equivalence_class(self, index_set=None):
                  ([[1, 3], [2, 4]], [[1, 2], [3, 4]], 3)]
                 sage: T = crystals.Tableaux(['A',4], shape=[3,2])
                 sage: G = T(2,1,4,3,5).dual_equivalence_class()
-                sage: G.edges(sort=True)
-                [([[1, 3, 5], [2, 4]], [[1, 3, 4], [2, 5]], 4),
-                 ([[1, 3, 5], [2, 4]], [[1, 2, 5], [3, 4]], 2),
-                 ([[1, 3, 5], [2, 4]], [[1, 2, 5], [3, 4]], 3),
-                 ([[1, 3, 4], [2, 5]], [[1, 2, 4], [3, 5]], 2),
-                 ([[1, 2, 4], [3, 5]], [[1, 2, 3], [4, 5]], 3),
-                 ([[1, 2, 4], [3, 5]], [[1, 2, 3], [4, 5]], 4)]
+                sage: G.edges(sort=True,sort_vertices=True)
+                [([[1, 3, 4], [2, 5]], [[1, 3, 5], [2, 4]], 4), ([[1, 3, 4], [2, 5]], [[1, 2, 4], [3, 5]], 2), ([[1, 2, 5], [3, 4]], [[1, 3, 5], [2, 4]], 2), ([[1, 2, 5], [3, 4]], [[1, 3, 5], [2, 4]], 3), ([[1, 2, 3], [4, 5]], [[1, 2, 4], [3, 5]], 3), ([[1, 2, 3], [4, 5]], [[1, 2, 4], [3, 5]], 4)]
```

Comment:
please put this output on multiple lines

Reply:
done in e22d5f1f335d23fcdc5e6d7df40d35750339634e